files/settings-default.cfg

```diff
@@ -200,6 +200,9 @@ best attack = false
 # Difficulty.  Expressed as damage dealt and received. (e.g. -100 to 100).
 difficulty = 0
 
+#The maximum range of actor AI, animations and physics updates.
```

Comment:
Minor nitpick: this line needs a space between the number sign and The word to be style-consistent.

Reply:
Got it.